database/index.js

```diff
@@ -45,13 +51,31 @@ let save = (reviews) => {
   return Promise.all(savePromises);
 }
 
-// fetch 25 things
-let fetch = () => {
-  return Review.find().sort('_id').limit(25);
+
+let fetchReviews = (callback) => {
+  console.log('fetchReviews invoked! Serving you 10 reviews 😀');
+  Review.find(null, null, {
+    limit: 10
```

Comment:
Looks great. In the future, it would be nice to use random value for limit. (Math.floor(Math.random() * 20) + 3)

Reply:
Good point. Will do, thank you!!